tests/syscall/fs/fs.edl

```diff
@@ -3,12 +3,12 @@
 
 enclave {
 
-    //from "sgxfs.edl" import *;
-
     trusted {
         public void test_fs(
             [string, in] const char* src_dir,
             [string, in] const char* tmp_dir);
-
+        public void test_fs_linux(
```

Comment:
I see this was added to an EDL that was moved out of the linux subdirectory, but this is linux specific isn't it?
Until we have preprocessor directives as in issue #2833, we may want to keep separate EDL files.
@jhand2 do you have a recommendation here?

Reply:
Yes this is linux specific. To match the host side we should use two separate edl files, but enclave side it is hard to make corresponding functions linux specific. Maybe I can update in next pr.